omniqueue/src/backends/redis/mod.rs

```diff
@@ -85,6 +85,63 @@ impl RedisConnection for RedisClusterConnectionManager {
     }
 }
 
+// First element is the raw payload slice, second
+// is `num_receives`, the number of the times
+// the message has previously been received.
+type InternalPayload<'a> = (&'a [u8], usize);
+
+// The same as `InternalPayload` but with an
+// owned payload.
+type InternalPayloadOwned = (Vec<u8>, usize);
```

Comment:
Wdym by these being less memory efficient when they're `struct`s?

Reply:
Structs would be fine. The issues were related to making the new functions methods of the struct.

Reply:
I went ahead and made them structs -- seems a bit more clear :shrug: